scripts/qa_crowbarsetup.sh

```diff
@@ -4455,6 +4455,27 @@ function onadmin_reapply_openstack_proposals
     done
 }
 
+function is_precheck_passed
+{
+    precheck_output=$1
+    required=false
+    for line in $precheck_output; do
+        if $required; then
```

Comment:
yes, we only want to complain about the required ones.

About the `[[ ]]`, not sure they are necessary because I tested that without them.